src/sass/claims-status.scss

```diff
@@ -1048,3 +1061,13 @@ $marker-text-width: 9rem;
     }
   }
 }
+
+.closed-appeal-notice {
+  line-height: 40px;
+  text-align: center;
+  font-family:  "Source Sans Pro", sans;
```

Comment:
I don't know if this is specifically specced anywhere, but I've seen hierarchy used in some `/common` components:
`"Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif;`

Reply:
Oh brilliant! I meant to revisit this to find what that hierarchy should be. Thanks!